src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -641,8 +641,60 @@ available on the [Fuchsia devsite].
 
 ### Running the compiler test suite
 
-Running the Rust test suite on Fuchsia is [not currently supported], but work is
-underway to enable it.
+Pre-requisites for running the Rust test suite on Fuchsia are:
+1. Installation of `rustc` via "[Targeting Fuchsia with a compiler built from source](#targeting-fuchsia-with-a-compiler-built-from-source)", including `config-env.sh` and `config.toml`.
+1. Checkout of Rust source.
+1. Download of the Fuchsia SDK. Minimum supported SDK version is [4xjxrGUrDbQ6_zJwj6cDN1IbWsWV5aCQXC_zO_Hu0XkC](https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core/linux-amd64/+/4xjxrGUrDbQ6_zJwj6cDN1IbWsWV5aCQXC_zO_Hu0XkC)
+
+Interfacing with the Fuchsia emulator is handled by our test runner script located
+at `${RUST_SRC_PATH}/src/ci/docker/scripts/fuchsia-test-runner.py`.
+
+We start by activating our Fuchsia test environment. From a terminal:
+
+**Issue command from ${RUST_SRC_PATH}**
+```sh
+src/ci/docker/scripts/fuchsia-test-runner.py start
+    --rust ${RUST_SRC_PATH}
```

Comment:
Could be `.` if the script supports it and then you can remove mention of `${RUST_SRC_PATH}` altogether. Rust developers would expect to be running tests from this directory anyway and putting it in an env variable is slightly inconvenient.

Reply:
Confirmed this works -- pushing adjustment now